v20.1/enable-node-map.md

```diff
@@ -8,21 +8,23 @@ toc: true
 On a secure cluster, this area of the Admin UI can only be accessed by an `admin` user. See [Admin UI access](admin-ui-overview.html#admin-ui-access).
 {{site.data.alerts.end}}
 
-The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
+The **Node Map** visualizes the geographical configuration of a multi-regional cluster by plotting the node localities on a world map. The Node Map also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance.
```

Comment:
I think this would be easier to read as something list-y, e.g.,

The node map is useful for:
- visualizing the geographic config of a multi-region cluster on a world map
- viewing real-time cluster metrics
- drilling down to individual nodes for monitoring health & performance

... or similar